index.bs

```diff
@@ -3759,15 +3759,6 @@ WebAuthn [=[RP]=].
     Specification](https://dev.w3.org/geo/api/spec-source.html#coordinates_interface).
 
     <pre>
-    CDDL:
-      {
-       loc: positionCoordinates,
-      }    
-      positionCoordinates = [
-         latitude: float64,
-         longitude: float64,
-         altitude: float64
-         ]
```

Comment:
An unrelated change appears to have been made in this PR.  Please remove this extraneous change, as it will cause merge conflicts with other PRs.